O/onnxruntime/build_tarballs.jl

```diff
@@ -0,0 +1,109 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "onnxruntime"
+version = v"1.10.0"
+
+# Collection of sources required to complete build
+onnxruntime_source = GitSource("https://github.com/microsoft/onnxruntime.git", "0d9030e79888d1d5828730b254fedc53c7b640c1"; unpack_target = "onnxruntime_clone")
+clone_recursive_path = "onnxruntime"
+clone_recursive_script = """
+rm -rf $(clone_recursive_path)
+git clone $(onnxruntime_source.url) $(clone_recursive_path)
+cd $clone_recursive_path
+git checkout $(onnxruntime_source.hash)
+git submodule update --init --recursive --jobs 8
+cd ..
+"""
+run(`bash -c "$clone_recursive_script"`)
+source_sources = [
+    onnxruntime_source,
+    DirectorySource(clone_recursive_path)
+]
+
+# Bash recipe for building across all platforms
+source_script = raw"""
+cd $WORKSPACE/srcdir
+
+wget https://github.com/Kitware/CMake/releases/download/v3.22.1/cmake-3.22.1-linux-x86_64.sh
+chmod u+x cmake-3.22.1-linux-x86_64.sh 
+./cmake-3.22.1-linux-x86_64.sh --skip-license --include-subdir
+CMAKE=`pwd`/cmake-3.22.1-linux-x86_64/bin/cmake
+
+wget https://github.com/protocolbuffers/protobuf/releases/download/v3.16.1/protoc-3.16.1-linux-x86_64.zip
```

Comment:
What's wrong with https://github.com/JuliaPackaging/Yggdrasil/blob/4ea0a2c9773dbe743abd10f9bd7567a6046ea01b/P/protoc/build_tarballs.jl?

Reply:
It needs a specific version of `protoc` (3.16): https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=16855&view=logs&j=2125d72a-77c5-58f3-9425-89906e3f565e&t=6a65a92e-fb28-56c0-7742-a34594e36aa2

Reply:
There you go: https://github.com/JuliaRegistries/General/pull/54008

Reply:
Am I waiting on a cache of the General registry?
https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=16909&view=logs&j=2125d72a-77c5-58f3-9425-89906e3f565e&t=6a65a92e-fb28-56c0-7742-a34594e36aa2&l=29

Reply:
[Yeah](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=16909&view=logs&j=96c13d9d-dfd2-51e5-2f76-4a0b3ce09abc&t=09532434-8b0a-5f0e-118d-a3181cde5ec8&l=26)
```
[ Info: PkgServer: https://us-east.pkg.julialang.org/registries.conservative -- last updated: Sun, 06 Feb 2022 14:59:46 GMT (49 minutes, 52 seconds ago)
```

Reply:
Seems like it got fixed.